### ISSUE TITLE
Confusing example in readme

### ISSUE THREAD

**Issue by Haroenv** [olegskl/gulp-stylelint#20]
Usually you'd put your required packages before using them, and thus would seem this:

```js
const gulp = require('gulp');
const gulpStylelint = require('gulp-stylelint');

gulp.task('lint-css', function lintCssTask() {
  return gulp
    .src('src/**/*.css')
    .pipe(gulpStylelint({
      reporters: [
        {formatter: 'string', console: true}
      ]
    }));
});
```

clearer than what it is now:

```js
gulp.task('lint-css', function lintCssTask() {
  const gulp = require('gulp');
  const gulpStylelint = require('gulp-stylelint');

  return gulp
    .src('src/**/*.css')
    .pipe(gulpStylelint({
      reporters: [
        {formatter: 'string', console: true}
      ]
    }));
});
```

**Comment by olegskl** [comment]
Hello and thanks for noticing this.

There is actually a performance reason requiring dependencies inside gulp tasks. In a common use case there are many gulp tasks, each using its own set of dependencies. When you require those dependencies inside tasks, only the necessary dependencies will be lazily required and resolved, shaving off some startup time.

There is no reason, however, to require gulp inside of a gulp task. I guess I was drunk or sleepy.

**Comment by Haroenv** [comment]
So something in between makes most sense

**Comment by olegskl** [comment]
This issue has been fixed and released in 2.0.1, so I'm closing it. Feel free to reopen if necessary.
